pxtblocks/fields/field_userenum.ts

```diff
@@ -21,6 +21,18 @@ namespace pxtblockly {
         }
 
         doClassValidation_(value: any) {
+            // The format of the name is 10mem where "10" is the value and "mem" is the enum member
+            if (this.sourceBlock_ && this.sourceBlock_.workspace) {
```

Comment:
```suggestion
            if (this.sourceBlock_?.workspace) {
```

tiny nit, we tend to prefer safe navigation operator nowadays for ease of reading~